modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1906,6 +1906,42 @@ TEST_P(SizeRTest, ParseTest)
     EXPECT_EQ(out_sz, sz);
 }
 
+namespace {
+    class TestMediaBGR final : public cv::MediaFrame::IAdapter {
+        cv::Mat m_mat;
+        using Cb = cv::MediaFrame::View::Callback;
+        Cb m_cb;
+
+    public:
+        explicit TestMediaBGR(cv::Mat m, Cb cb = []() {})
+            : m_mat(m), m_cb(cb) {
+        }
+        cv::GFrameDesc meta() const override {
+            return cv::GFrameDesc{ cv::MediaFormat::BGR, cv::Size(m_mat.cols, m_mat.rows) };
+        }
+        cv::MediaFrame::View access(cv::MediaFrame::Access) override {
+            cv::MediaFrame::View::Ptrs pp = { m_mat.ptr(), nullptr, nullptr, nullptr };
+            cv::MediaFrame::View::Strides ss = { m_mat.step, 0u, 0u, 0u };
+            return cv::MediaFrame::View(std::move(pp), std::move(ss), Cb{ m_cb });
+        }
+    };
+};
+
+TEST_P(SizeMFTest, ParseTest)
+{
+    cv::GFrame in;
+    cv::Size out_sz;
+    int counter = 0;
+    cv::Mat bgr = cv::Mat::eye(sz.height, sz.width, CV_8UC3);
+    cv::MediaFrame frame = cv::MediaFrame::Create<TestMediaBGR>(bgr, [&counter]() {counter++; });
+
+    auto out = cv::gapi::streaming::size(in);
```

Comment:
Could you please move `cv::GFrame` declaration a bit down to the 1937 line?

Reply:
Done